### DISCUSSION TITLE
Failed to write logs into GCS

### DISCUSSION THREAD

**Discussion by ginolegigot** [apache/airflow#26976] [Q&A]
Hello,
I think I properly read https://airflow.apache.org/docs/apache-airflow-providers-google/stable/logging/gcs.html so i configured [logging] as following:
```cfg
[logging]
base_log_folder = 

remote_logging = True

remote_log_conn_id = 

google_key_path = 

remote_base_log_folder = gs://bucket_name/airflow_logs
```
I installed apache airflow on 2.4.1 Celery VM which are in a small cluster for resiliency.
I'm struggling to configure airflow so that it can send logs into GCS, I actually noticed i'm not able to properly configure these parameters:
- [scheduler] child_process_log_directory
- [webserver] error_logfile and access_logfile
- [logging] dag_processor_manager_log_location

I tried for these parameters a basic relative folder path like:
child_process_log_directory = "scheduler"
Or:
child_process_log_directory = gs://bucket_name/airflow_logs/scheduler
which is somehow interpolated in the logs as: gs:/bucket_name/airflow_logs/scheduler

But none are working. Ah and i can successfully from my vm do a gsutil rm -r a folder on the bucket or list the objects inside so i think i'm OK with the roles given to my VM on this bucket.

I get these logs:
FileNotFoundError: [Errno 2] No such file or directory: 'scheduler/2022-10-10'
PermissionError: [Errno 13] Permission denied: 'scheduler'

An idea ?

**Comment by potiuk** [comment]
Impossible to hlpe with the limited info. There is no information at all about your deployment, Seems like you have permission problems, directlyr folders to create etc. so hard to say what's wrong. I guess you will need to see exactly what file access operations are happening and what permissions are missing.
